.run/Debug tests.run.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
I'm not sure there's `.run/*` files should be version controlled?

Reply:
@litlepeathis will greatly simplify the life of those who will develop in PyCharm.  If this is a problem, I can remove it.

Reply:
No, it's fine